src/types/types.js

```diff
@@ -301,7 +301,6 @@ export type EdgeCurrencyInfo = {
   // Chain information:
   canAdjustFees?: boolean, // Defaults to true
   canImportKeys?: boolean, // Defaults to false
-  canReplaceByFee?: boolean, // Defaults to false
```

Comment:
We need to keep this flag, since we will use it to select between “pending” and “this transaction may be cancelled” in the GUI. So, it is no longer about *doing* RBF, but rather about whether RBF might be done to us.

Reply:
Are you saying that this flag is to signal to the GUI whether a particular wallet supports RBF on transactions that may be broadcast, and this is useful in order to detect this behavior in the GUI before sending a transaction? Whether a particular transaction is replaceable is still determined by the `accelerate()` method, however this field in the currency info can still provide the GUI with information about the capabilities of the particular blockchain underpinning the wallet.

Reply:
No, it's about *receiving* money, from other people. Do we trust the sender, or not?

Reply:
Dropped the commit which was removing the `canReplaceByFee` field